packages/app/components/drop/drop-select.tsx

```diff
@@ -56,9 +84,13 @@ export const DropSelect = () => {
             }
             description="Promote your latest music: give your fans a free collectible for saving your song to their library."
             ctaLabel={
-              canCreateMusicDrop ? "Create Music Drop" : "Request Access"
+              canCreateMusicDrop ? "Create a Music Drop" : "Request Access"
             }
             onPress={() => {
+              if (!canUserCreateEditions) {
```

Comment:
AFAIK music drop should still be free for the verified account, if non-verified should be linked to the [Typeform](https://form.typeform.com/to/pXQVhkZo) to apply it.

Reply:
good catch 🙏! Fixed